src/box.jl

```diff
@@ -18,6 +18,9 @@ Box(c::SVector{N}, d::SVector{N},
 Box(c::AbstractVector, d::AbstractVector, axes=eye(length(c)), data=nothing) =
     (N = length(c); Box(SVector{N}(c), SVector{N}(d), SMatrix{N,N}(axes), data))
 
+Box(b::NTuple{2,AbstractVector}, axes=eye(length(b[1])), data=nothing) =
```

Comment:
I see.  But what if both construction methods (one specifying `c` and `d` and the other specifying `b`) are expected to be nearly equally popular among users?  In that case, wouldn't users want to use the most representative function name (i.e., `Box`) according to their preferences?

I understand that in the present case confusion could arise, because `c`, `d`, `b1`, `b2` are all vectors with the same length.  At the same time, though, I thought users would relatively easily get the difference: because `b1` and `b2` are the same kind (both are corners) where as `c` and `d` are not (`c` is a point and `d` is lengths), two vectors passed as one tuple would mean `(b1, b2)` rather than `(c, d)`.

What do you think?  If you still think this would be very confusing to users,
- I can implement `boxcorners` as you suggested (if you think it would be a useful constructor function), or
- I will be fine with simply closing this PR, because users who prefer `Box` specifying `b` (like me) can always define it themselves outside `GeometryPrimitives`.